.tekton/cli-main-ci-pull-request.yaml

```diff
@@ -100,11 +100,11 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
```

Comment:
Why change the defaults? Set the parameter in the `spec.params` above

Reply:
Oh yeah, thanks.